single-line-diagram/single-line-diagram-core/src/main/java/com/powsybl/sld/layout/GraphRefiner.java

```diff
@@ -23,10 +23,12 @@
 public class GraphRefiner {
     private final boolean removeUnnecessaryFictitiousNodes;
     private final boolean substituteSingularFictitiousByFeederNode;
+    private final boolean removeFictitiousSwitchNode;
```

Comment:
In a way, it makes sense to put that parameter here, but how is it possible for the user to change this easily? It would be easier to put it in LayoutParameters. Does it make sense? I'd say yes

Reply:
Isn't it ok to do:

```
 factory = new PositionVoltageLevelLayoutFactory();
 factory.setRemoveFictitiousSwitchNodes(true);
```

And then use this factory to draw the diagram?

Reply:
I finally put it in LayoutParameters, it is indeed more user-friendly that way. It would be good too to think about moving other parameters from the PositionVoltageLevelLayoutFactory to the LayoutParameters. 
I created an issue for that #503.